.travis.yml

```diff
@@ -20,8 +20,20 @@ matrix:
             - gcc-4.9
             - g++-4.9
       env: DEVITO_ARCH=gcc-4.9 DEVITO_OPENMP=0 INSTALL_TYPE=pip_setup RUN_EXAMPLES=False MPI_INSTALL=1
-    - os: linux
-      python: "3.6"
+    - python: 3.7
```

Comment:
is this a new build??

Reply:
Yes, now two builds, one for 3.6 and one for 3.7

Reply:
I don't think a whole new build is a good idea, as this will increase the turnaround time by at least another hour (maybe two). Can we not use 3.6 in one of the existing builds?

Reply:
Well no, this test python 3.6 and 3.7 with pip then there is the conda builds. Can try to merge one of the conda with one of the two pip but will reduce conda testing. I'm not sure about turnaround time as they supposed to run toghether so don't see why it would be slower

Reply:
Removed one, added one to pipelines instead as it is faster